plugins/modules/rosa_cluster.py

```diff
@@ -229,8 +229,9 @@ def run_module():
     # check that rosa is minimum version
     # MIN_ROSA_VERSION
     rosa_version_cmd = [rosa, "version"]
+    # rosa_version_cmd = [rosa, "version", "|", "head", "-1"]
     rc, stdout, stderr = module.run_command(rosa_version_cmd)
-    rosa_version = stdout.rstrip()
+    rosa_version = stdout.rstrip()[:7]
```

Comment:
I think this makes more sense `stdout.rstrip().split('\n')[0]`

Reply:
updated